odoo/addons/base/models/ir_translation.py

```diff
@@ -27,15 +27,15 @@ class IrTranslationImport(object):
     """
     _table = 'tmp_ir_translation_import'
 
-    def __init__(self, model):
+    def __init__(self, cr, table, overwrite=False):
```

Comment:
I never understood the point of being generic here.
The value of `table`  is *always* `'ir_translation'`.
I would simply remove it from the parameters.
```suggestion
    def __init__(self, cr, overwrite=False):
```